packages/components/src/Autocomplete/Autocomplete.test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
```

Comment:
We don't actually test the debounce in this file, as far as I can see. Am I missing it?

Reply:
I have added a backlog item to fix this test.